README.md

```diff
@@ -1,3 +1,8 @@
+Eclipse SmartHome now hosts the Orange implementation of a standard OSGi EnOcean base driver.

```

Comment:
I would reformulate it to:
"This repository hosts the reference implementation for the OSGi EnOcean Device Service specification, which is a sub-project of [Eclipse SmartHome](https://www.eclipse.org/smarthome/) that has been originally implemented and contributed by Orange."